nodewatcher/modules/monitor/http/survey/processors.py

```diff
@@ -0,0 +1,114 @@
+import datetime
+
+from django.utils.translation import gettext_noop as _
+from django.utils import timezone
+from django_datastream import datastream
```

Comment:
Group imports correctly. `django_datastream` is not Django and it is a third party module. Order: core modules, Django modules, nodewatcher modules, and 3rd party modules.